### PR TITLE
Move the bug submission process to the FAQ. Fixes #5502.

### LINKED ISSUE #5502: Web Bug Bounty FAQ are missing instructions on submitting reports
I was looking at https://www.mozilla.org/en-US/security/bug-bounty/faq-webapp/, trying to find a way to submit a report. "Once I have found a vulnerability, what next?" links nowhere, this section was removed in the https://github.com/mozilla/bedrock/commit/e4dd27a661a8d8e551074b9e18476bd138184d23 commit by @april. The necessary information exists under https://www.mozilla.org/en-US/security/web-bug-bounty/#how-to-submit-bugs but there is currently no link there. One *could* get there by clicking "What are the bounty payouts?" or "What types of issues will be considered as part of the bounty program?" but quite frankly - that is far from obvious.

### PR DESCRIPTION
**Pull request by april**
## Description
Moves the "How To Submit Bugs" to the FAQ and then adds a couple links to it in the top level Bug Bounty entry form.

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/5502

## Testing
Ran Bedrock locally, all links seem to work correctly. No display issues or anything.

@claudijd, could I get some quick r? on this? Thanks!

### CODE REVIEW

File: bedrock/security/templates/security/bug-bounty/faq-webapp.html [hunk 1]
```diff
@@ -97,9 +97,35 @@ <h2>Eligible bugs</h2>
...
       </dl>
 
-      <h2>Bug reporting</h2>
+      <h2 id="bug-reporting">Bug reporting</h2>
 
       <dl class="faq">
+        <dt id="what-next">Once I have found a vulnerability, what next?</dt>
+        <dd>
+          <p>The sooner we can reproduce and fix the bug, the sooner we can
+            fix it and send you your bounty. Please submit all bugs through
```
Comment (claudijd) [inline bedrock/security/templates/security/bug-bounty/faq-webapp.html]: " and send you your bounty" seems to imply all bugs get a bounty or least is something some might argue.  Maybe it's not all that important, but making note of the wording.

File: bedrock/security/templates/security/bug-bounty/faq-webapp.html [hunk 2]
```diff
@@ -97,9 +97,35 @@ <h2>Eligible bugs</h2>
...
+            the <a href="https://bugzilla.mozilla.org/form.web.bounty">Bugzilla web bounty form</a>.
+            <strong>Do not send vulnerabilities via email and please avoid using video.</strong></p>
```
Comment (claudijd) [inline bedrock/security/templates/security/bug-bounty/faq-webapp.html]: Joke: " ... using video or we will find you" ;)

Comment (april) [reply bedrock/security/templates/security/bug-bounty/faq-webapp.html]: "We know who you are."

File: bedrock/security/templates/security/bug-bounty/faq-webapp.html (hunk 1 above)
Comment (claudijd) [reply bedrock/security/templates/security/bug-bounty/faq-webapp.html]: it's fine as is

Comment (jpetto) [reply bedrock/security/templates/security/bug-bounty/faq-webapp.html]: This reads a little odd, as "fixing" is mentioned twice:

>The sooner we can reproduce and fix the bug, the sooner we can fix it...

File: bedrock/security/templates/security/bug-bounty/faq-webapp.html [hunk 3]
```diff
@@ -97,9 +97,35 @@ <h2>Eligible bugs</h2>
...
+
+          <p>There are three main things you can provide which will help us
+            to evaluate your submission quickly and pay a bounty sooner:</p>
+
+          <ol>
+            <li><a href="{{ url('security.bug-bounty.faq-webapp') }}#attack-scenario">What is the attack scenario?</a></li>
+            <li><a href="{{ url('security.bug-bounty.faq-webapp') }}#step-by-step-exploit">What is the step-by-step exploit process?</a></li>
```
Comment (jpetto) [inline bedrock/security/templates/security/bug-bounty/faq-webapp.html]: Nit-pick here, but no need for the full URL if we're just linking to an in-page anchor.

Comment (april) [reply bedrock/security/templates/security/bug-bounty/faq-webapp.html]: Good call, I forgot to clean that up when I moved this text from a different page.

**Comment by april** [comment]
All review items should be address.  :)

**Comment by jpetto** [comment]
Looks good! Can you squash the commits?

**Pull request merged**